src/main/scala/com/typesafe/conductr/sbt/ConductR.scala

```diff
@@ -25,25 +26,36 @@ private[conductr] object ConductR {
   final val DefaultConductrProtocol = "http"
   final val DefaultConductrHost = "127.0.0.1"
   final val DefaultConductrPort = 9005
+
   val conductrAttrKey = AttributeKey[ActorRef]("sbt-conductr")
   val actorSystemAttrKey = AttributeKey[ActorSystem]("sbt-conductr-actor-system")
 
-  def loadBundle(bundle: URI, config: Option[URI], loadTimeout: Timeout, state: State): String = {
+  def loadBundle(pv: String, bundle: URI, config: Option[URI], loadTimeout: Timeout, state: State): String = {
```

Comment:
`protocolVersion` instead of `pv` (Naming should be consistent across the project).